plotly/tests/test_core/test_tools/test_make_subplots.py

```diff
@@ -143,18 +143,10 @@ def test_two_row_bottom_left(self):
         expected = Figure(
             data=Data(),
             layout=Layout(
-                xaxis1=XAxis(
-                    domain=[0.0, 1.0],
-                    anchor='y1'
-                ),
```

Comment:
Why is this removed?

Reply:
It's a mistake. Removing it  produced the same `expected` subplot, but I changed anchor to `y1` and `xaxis1` to `xaxis` and everything works as expected